src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -145,8 +145,11 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
             const controllers = await this.createNotebookControllers(connections);
 
             // Send telemetry related to fetching the kernel connections
-            // KERNELPUSH: undefined works for telemetry?
-            sendNotebookControllerCreateTelemetry(undefined, controllers, stopWatch);
+            sendKernelListTelemetry(
+                Uri.file('test.ipynb'), // Give a dummy ipynb value, we need this as its used in telemetry to determine the resource.
```

Comment:
Would it be better  ```'if undefined use 'test.ipynb'```  was in the ```sendKernelListTelemetry```? It seems like it would know better that undefined is not supported.

Reply:
I don't want to do that, as we have plenty of places in interactive window where we end up with `undefined` for the `resource`.
I.e. this is backwards compatible, `resource` can be undefined & only in the case of interactive window (with notebooks we always have ipynb).